ansible_collections/arista/avd/docs/plugins/Modules and action plugins/validate_and_template.md

```diff
@@ -0,0 +1,56 @@
+# validate_and_template
+
+!!! note
+    Always use the FQCN (Fully Qualified Collection Name) `arista.avd.validate_and_template` when using this plugin.
+
+Validate input data according to Schema\, render Jinja2 template and write result to a file\.
+
+## Synopsis
+
+The \`arista\.avd\.validate\_and\_template\` Action Plugin performs data conversions and validation according to the supplied Schema\.
```

Comment:
Present in multiple docs, I'm debating if the synopsis should be one main paragraph vs a row of sentences.

Reply:
If the description is a list in the source files, it will be renderes as seperate paragraphs. So we should change the source files to use multiline strings if we want the output to be a single paragraph.